### PR TITLE
CXF-97346 Update YML files with exact Names

### PR DESCRIPTION
**Pull request by jkallem-equinix**

### CODE REVIEW

File: .github/workflows/terratests-virtualdevice-prod-suite.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: terratests-prod-suite
+name: terratests-vurtualdevice-prod-suite
```
Comment (thogarty) [inline .github/workflows/terratests-virtualdevice-prod-suite.yaml:1]: Still have a typo in this one.

**Review by thogarty — CHANGES_REQUESTED** [review]
See comment.

File: .github/workflows/terratests-virtualdevice-prod-suite.yaml (hunk 1 above)
Comment (jkallem-equinix) [reply .github/workflows/terratests-virtualdevice-prod-suite.yaml]: Updated with right name.

**Review by thogarty — APPROVED** [review]
LGTM

**Pull request merged**